packages/safety-suite-api/src/api/models/types.ts

```diff
@@ -45,18 +45,47 @@ export interface Model<F, R, C> {
 }
 
 export type SearchText = {
+  /**
+   * Field or computation name.
+   */
   field: string;
```

Comment:
Are relations supported here? If so then this could also be a string[]. I don't think that they are, but wanted to verify.

Reply:
Doesn't look like that's supported.
![image](https://user-images.githubusercontent.com/13290954/109702494-4f288880-7b62-11eb-9705-90e9167b21e1.png)